source/features/comment-shortcuts.tsx

```diff
@@ -0,0 +1,69 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {isEditable} from '../helpers/dom-utils';
+
+const shortcutClass = new Map<string, string>([
+	['e', '.unminimized-comment [aria-label="Edit comment"]:not([hidden])'],
+	['d', '[aria-label="Delete comment"]']
+]);
+
+function runShortcuts(event: KeyboardEvent): void {
+	const focusedComment = select(':target');
+	if (isEditable(event.target) || !focusedComment) {
+		return;
+	}
+
+	if (['j', 'k'].includes(event.key)) {
+		event.preventDefault();
+
+		const items = select.all('.js-minimizable-comment-group')
+			.filter(comment => !comment.querySelector('.minimized-comment:not(.d-none)'));
```

Comment:
Note to self: verify these selectors

Reply:
https://github.com/sindresorhus/refined-github/blob/0fa514dbeeab6395ef095cff80beac41443a6574/source/features/preview-hidden-comments.tsx#L13

Reply:
How about this?
```suggestion
		const items = select.all('.js-targetable-element');
```

Reply:
This selector also works on on PR files, so I'd rename the feature to `keyboard-navigation`

Reply:
But we still need to filter out the minimized comments. 
IMHO opening a minimized comment is counter productive

What do you think of my idea https://github.com/sindresorhus/refined-github/pull/3242#issuecomment-647141375?

Reply:
I think feature should (eventually, if not now) be able to unhide those comments as well, or at least expand them.

Perhaps you should still use that “targetable” selector and run the filter to exclude what we don’t want (e.g. hidden comments, at the moment)

Reply:
When using the tagetable selector all the pr files are included in "items".

From my understanding of what you want is: `j` `k` should not select files only `o` and `v`

Reply:
I specifically said that I want the files to be selected as well. O and V will be additional shortcuts to toggle and mark files as viewed. That part can come later, but even just navigation is useful.